src/app/tasks/checker-inbox-and-tasks-tabs/client-approval/client-approval.component.ts

```
@@ -174,7 +175,7 @@ export class ClientApprovalComponent {
       response.forEach((responseEle: any) => {
         if ((responseEle.statusCode = '200')) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)